### ISSUE TITLE
Possibility to show HTML/PHP as a tile

### ISSUE THREAD

**Issue by agc346** [VA3HDL/hamdashboard#12]
Is it possible to point directly at an HTML or PHP page such as HamClock or WeeWX to get real time data on the dashboard?

**Comment by VA3HDL** [comment]
This is already in the To Do list, along with the separate rotate timer for each tile (now all tiles rotate every 30 s)

**Comment by midair55** [comment]
Do you now regret starting this “simple” project that has become a job?

**Comment by VA3HDL** [comment]
Not at all. On the contrary I feel I made a contribution to the community for the ones that found a valuable addition to their shacks. I received very good feedback from many and that keeps me going.

Thanks for your help.

**Comment by midair55** [comment]
Hmmmm.......  The  graphical weather PHP image mentioned above is not refreshing with, "<meta http-equiv="refresh" content="1800">" in the HTML.  Go big deal..

**Comment by VA3HDL** [comment]
I believe because the time on the chart is embedded on the links you used...

https://forecast.weather.gov/meteograms/Plotter.php?lat=28.3874&lon=-80.6048&wfo=MLB&zcode=FLZ747&gset=18&gdiff=3&unit=0&tinfo=**EY5**&ahour=0&pcmd=11101111110000000000000000000000000000000000000000000000000&lg=en&indu=1!1!1!&dd=&bw=&hrspan=48&pqpfhr=6&psnwhr=6
https://forecast.weather.gov/meteograms/Plotter.php?lat=28.3893&lon=-80.6088&wfo=MLB&zcode=FLZ447&gset=18&gdiff=3&unit=0&tinfo=**EY6**&ahour=0&pcmd=1111111111&lg=en&indu=1!1!1!&dd=3&bw=0&hrspan=48
https://forecast.weather.gov/meteograms/Plotter.php?lat=28.3893&lon=-80.6088&wfo=MLB&zcode=FLZ447&gset=18&gdiff=3&unit=0&tinfo=**EY7**&ahour=0&pcmd=1111111111&lg=en&indu=1!1!1!&dd=3&bw=0&hrspan=48

the part in bold is the hour of the chart

**Comment by midair55** [comment]
Hit - https://forecast.weather.gov/meteograms/Plotter.php?lat=28.3893&lon=-80.6088&wfo=MLB&zcode=FLZ447&gset=18&gdiff=3&unit=0&tinfo=EY6&ahour=0&pcmd=1111111111&lg=en&indu=1!1!1!&dd=3&bw=0&hrspan=48

Then wait until the top of the hour and refresh.  The X axis time-line changes by one hour where the first labeled hour is the next hour.  I don't recall what values I had to change so it shows the 'next' hour labeled minus two hours.   The 'hrspan' is the number of hours to display.
